plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -960,7 +963,17 @@ def create_vm(self, vmid, newid, node, name, memory, cpu, cores, sockets, update
                 kwargs.update(kwargs[k])
                 del kwargs[k]
 
-        # Rename numa_enabled to numa. According the API documentation
+        # Map bool to string, according to API documentation.
+        try:
+            if boolean(kwargs['agent'], strict=True):
+                kwargs['agent'] = 'enabled=1'
+            else:
+                kwargs['agent'] = 'enabled=0'
```

Comment:
Is a JSON string `enabled=0` equivalent to a JSON `false` value for all supported versions of proxmoxer / the API? (The same for `enabled=1` and `true`.)

Reply:
I'm not quite sure. May be best to pass the boolean directly as before.